services/galley/test/integration/API/Teams/LegalHold.hs

```diff
@@ -607,7 +613,9 @@ withTestService mkApp go = do
     srv <- liftIO . Async.async $
         Warp.runTLS tlss defs $
             mkApp buf
-    go buf `finally` liftIO (Async.cancel srv)
+    recovering def [\_ -> Handler $ \(SomeException _) -> pure False]
```

Comment:
I'm a little confused, this says DON'T retry on ANY exceptions... not positive, but I think this does basically nothing?

I think we just want `recoverAll`?

Reply:
you're not nearly as confused as i am.  :)

also you're right.  fixing...